src/ape/api/providers.py

```diff
@@ -960,18 +966,15 @@ def send_transaction(self, txn: TransactionAPI) -> ReceiptAPI:
             if "nonce too low" in str(vm_err):
                 # Add additional nonce information
                 new_err_msg = f"Nonce '{txn.nonce}' is too low"
-                raise VirtualMachineError(
+                vm_err = VirtualMachineError(
```

Comment:
I think if the `Nonce` is too low, there won't be a receipt, so we maybe could raise here as normal

Reply:
Updated in cf803b8b